Thu Apr 14 10:38:48 EEST 2011  Sergei Trofimovich <slyfox@community.haskell.org>
  * build filepath and deepseq (convinces 'make install')
diff -rN -u old-jhc/Makefile.am new-jhc/Makefile.am
--- old-jhc/Makefile.am	2011-04-14 10:40:58.331101006 +0300
+++ new-jhc/Makefile.am	2011-04-14 10:40:58.333101006 +0300
@@ -342,7 +342,9 @@
 xhtml-3000.2.0.1.hl: lib/ext/xhtml.cabal base-1.0.hl jhc-1.0.hl
 	perl utils/build_extlibs.prl $<
 filepath-1.2.0.0.hl: lib/ext/filepath.cabal jhc-1.0.hl base-1.0.hl
+	perl utils/build_extlibs.prl $<
 deepseq-1.1.0.2.hl: lib/ext/deepseq.cabal jhc-1.0.hl base-1.0.hl applicative-1.0.hl containers-0.3.0.0.hl
+	perl utils/build_extlibs.prl $<
 
 JHC_EXT_LIBS = containers-0.3.0.0.hl Diff-0.1.2.hl html-1.0.1.2.hl HUnit-1.2.2.1.hl parsec-2.1.0.1.hl \
     pretty-1.0.1.1.hl QuickCheck-1.2.0.0.hl safe-0.2.hl smallcheck-0.4.hl utility-ht-0.0.5.1.hl \
